src/components/ResortsData.js

```diff
@@ -18,30 +20,34 @@ class ResortsData extends Component{
     this.fetchResorts();
   }
 
-  componentWillUpdate(nextProps, nextState) {
-    if (nextState.page !== this.state.page) {
+  componentDidUpdate(prevProps, prevState) {
+    if (prevState.page !== this.state.page) {
       this.fetchResorts();
     }
   }
 
   fetchResorts(){
     this.setState({isLoading: true})
-    fetch(`http://localhost:3001/resorts?_page=${this.state.page}&_limit=1`)
+    fetch(`http://localhost:3001/resorts?_page=${this.state.page}&_limit=${this.state.perPage}`)
         .then(response => {
           if (response.ok) {
-            return response.json();
+            return response.json().then((data) => ({data, headers: response.headers}))
           } else {
             throw new Error('Something went wrong ...');
           }
         })
-        .then(data => this.setState({ data: data, isLoading: false }))
+        .then(({data, headers}) => {
+          this.setState({ data: data,  pageCount: Math.ceil(headers.get('x-total-count')/ this.state.perPage), isLoading: false })
```

Comment:
(minor)
This line is quite long. Please split params to separate line.
Also, use shorthand for data.